draft-ietf-masque-connect-udp.md

```diff
@@ -45,119 +55,222 @@ document are to be interpreted as described in BCP 14 {{!RFC2119}} {{!RFC8174}}
 when, and only when, they appear in all capitals, as shown here.
 
 In this document, we use the term "proxy" to refer to the HTTP server that
-opens the UDP socket and responds to the CONNECT-UDP request. If there are
-HTTP intermediaries (as defined in Section 2.3 of {{RFC7230}}) between the
+opens the UDP socket and responds to the UDP proxying request. If there are
+HTTP intermediaries (as defined in {{Section 3.7 of SEMANTICS}}) between the
```

Comment:
As a note, I was finding that annotating sections this way was causing errors in the xml submission when trying to submit DATAGRAM.

Reply:
Apparently it's supposed to work if it's listed in the normative section at the top. I'll double check though